stripe/src/main/java/com/stripe/android/model/PaymentMethodCreateParams.kt

```diff
@@ -164,7 +164,8 @@ data class PaymentMethodCreateParams internal constructor(
         Sofort("sofort"),
         P24("p24"),
         Bancontact("bancontact"),
-        Giropay("giropay")
+        Giropay("giropay"),
+        EPS("eps")
```

Comment:
let's make this `Eps` to be consistent with the other types (e.g. FPX is `Fpx`)

Reply:
👍